packages/nova-react-test-utils/package.json

```
@@ -50,14 +102,14 @@
   "sideEffects": false,
   "access": "public",
   "publishConfig": {
-    "main": "./lib/index",
-    "types": "./lib/index.d.ts",
-    "module": "./lib/index.mjs",
+    "main": "./lib/relay/index",
+    "types": "./lib/relay/index.d.ts",
+    "module": "./lib/relay/index.mjs",
     "exports": {
       ".": {
-        "types": "./lib/index.d.ts",
-        "import": "./lib/index.mjs",
-        "require": "./lib/index.js"
+        "import": "./lib/relay/index.mjs",
+        "require": "./lib/relay/index.js",
+        "types": "./lib/relay/index.d.ts"
       }
```

I think you need both `relay` and `apollo` here to make them available for import? The way I understand "exports" to work is that it won't allow other paths to be accessed other than what is here.

```suggestion
       "./relay": {
         "import": "./lib/relay/index.mjs",
         "require": "./lib/relay/index.js",
         "types": "./lib/relay/index.d.ts"
       },
       "./apollo": {
         "import": "./lib/apollo/index.mjs",
         "require": "./lib/apollo/index.js",
         "types": "./lib/apollo/index.d.ts"
       }
      }
```

---

You might not have run into it because it's in `"publishConfig"`? But I'm not sure about that.

---

I basically used approach from https://github.com/testing-library/react-testing-library/blob/main/pure.js, just added .js and .mjs variants of these files

---

> And BTW I verified that using exports field forces us to change moduleResolution to Bundler 

I'm a bit confused about this one, that shouldn't be needed I think.  But I'm totally fine that we keep it this way.

---

Resolving then